README.rst

```diff
@@ -197,117 +281,9 @@ Here is an example of fetch a row:
     $dbh = null;
     echo "OK\n";
 
-Build and Test
+Running Tests For the PHP PDO Driver
```

Comment:
Test on Windows is different, please see the description here:
https://github.com/snowflakedb/pdo_snowflake/pull/226

Reply:
@SimbaGithub 

Could we add the doc changes to that PR so that the doc changes and the code changes are in the same PR?

Reply:
@SimbaGithub 

i.e. I'd rather not merge in doc changes for code changes that haven't yet been merged in.